bin/docserv-createconfig

```diff
@@ -168,7 +169,7 @@ def generate_config(languages, default_lang, git_url, branches, version="0.9.10"
         # etree.SubElement(extralinks, "link", href=FIXME).text = "SOME DESCRIPTION"
 
     tree = etree.ElementTree(root)
-    configfile = os.getcwd() + "/config.xml"
+    configfile = os.getcwd() + "/{}.xml".format(productid)
```

Comment:
IHMO, we shouldn't mix concatenating with "+" and the `.format()` method. There should be only one way to do it. As such, I'd suggest:

```suggestion
    configfile = "{}/{}.xml".format(os.getcwd(), productid)
```